gradle.properties

```
@@ -1,6 +1,6 @@
 org.gradle.caching=true
 group=io.ballerina.lib
-version=0.1.0-SNAPSHOT
+version=0.1.0
```

Lets keep the version as it is.
```suggestion
version=0.1.0-SNAPSHOT
```

---

> Lets keep the version as it is.

Thisaru asked to change the version.

---

Version should be a SNAPSHOT.

---

done [f06510f](https://github.com/ballerina-platform/module-ballerinax-hubspot.marketing.transactional/pull/3/commits/f06510f14866b80a439adafcebf61a523d2fd8b7)